src/components/Compare/Compare.style.tsx

```diff
@@ -382,7 +382,7 @@ export const Row = styled(TableRow)<{
   background-color: ${({ index, $headerRowBackground }) =>
     !isNumber(index)
       ? `${$headerRowBackground}`
-      : index % 2 === 0
+      : (index ?? 0) % 2 === 0
```

Comment:
Hrm... I'm not sure why this change was necessary.  `isNumber` is typed as:

```
        isNumber(value?: any): value is number;
```

So typescript's type narrowing should know that at this point in the code it is a number and you shouldn't need a `??` fallback.

Were you seeing something different? Am I confused?

Reply:
you are right, and I was confused by it as well, but next's typescript compiler was complaining, and this took care of it.  I can take it out for the time being, as it's unnecessary, but since it is still correct and works, I'm inclined to leave it as one less thing to cleanup later.